install/kots/manifests/gitpod-config-patch.yaml

```diff
@@ -4,5 +4,6 @@ metadata:
   name: gitpod-config-patch
   labels:
     app: gitpod
+    component: gitpod-installer
 data:
-  gitpod-config-patch.yaml: '{{repl if and (ConfigOptionEquals "advanced_mode_enabled" "1") (ConfigOptionNotEquals "config_patch" "") }}{{repl ConfigOption "config_patch" }}{{repl else }}{{repl printf "{}" | Base64Encode }}{{repl end }}'
\ No newline at end of file
+  gitpod-config-patch.yaml: '{{repl if and (ConfigOptionEquals "advanced_mode_enabled" "1") (ConfigOptionNotEquals "config_patch" "") }}{{repl ConfigOption "config_patch" }}{{repl else }}{{repl printf "{}" | Base64Encode }}{{repl end }}'
```

Comment:
nit: End with a new Line? 🤔

Reply:
I don't understand? It is ending with a new line